test/router/router.result

```diff
@@ -1347,6 +1347,67 @@ vshard.router.route(1):callro('echo', {'some_data'})
 - null
 - null
 ...
+-- gh-140: prohibit vshard.router/storage.cfg from multiple fibers.
+fiber_cfg_status = fiber.channel(2)
+---
+...
+_ = test_run:cmd("setopt delimiter ';'")
+---
+...
+function fiber_cfg(name) 
+	fiber.create(function()
+    	local status, err = pcall(vshard.router.cfg, cfg)
+		fiber_cfg_status:put({
+            name = name,
+            status = status,
+		})
+	end)
+end;
+---
+...
+_ = test_run:cmd("setopt delimiter ''");
+---
+...
+-- Fibers apply config one after another.
+fiber_cfg("f1")
+---
+...
+fiber_cfg("f2")
+---
+...
+fiber_cfg_status:get()
+---
+- name: f1
+  status: true
+...
+fiber_cfg_status:get()
+---
+- name: f2
+  status: true
+...
+-- Second fiber gets configuration timeout error.
+vshard.router.internal.errinj.ERRINJ_CONCURRENT_CFG_TIMEOUT = true
+---
+...
+vshard.consts.ROUTER_CFG_TIMEOUT = 1
```

Comment:
I propose you to drop vshard.consts.ROUTER/STORAGE_CFG_TIMEOUT. Because this really contradicts with how *.cfg works. In Tarantool *.cfg (box.cfg, swim.cfg, router.cfg, storage.cfg) never throws a timeout error. They may hang forever, may leave the instance in an orphan state, may return an error on a wrong configuration, but they don't throw a timeout error. I would like to keep it that way so as users would not need to check for timeouts.